src/test/converter/react/react.tsx

```diff
@@ -8,8 +8,10 @@ export interface DemoProps {
 export class Demo {
     private foo: number;
 
+    //@ts-ignore
```

Comment:
Necessary?

Reply:
I think it was complaining that `props` is never used.  I could add the same `this.foo` trick for `this.props`